resources/views/docs/mobile/2/getting-started/development.md

```diff
@@ -0,0 +1,201 @@
+
+We've tried to make this as straightforward as possible. Simply run:
+
+```shell
+php artisan native:patch-inertia
+```
+
+This will backup your current `vite.config.js` and replace it with one that 'fixes' Axios.
+
+You will just need to copy over any specific config (plugins etc) from your old Vite config to this new one.
+
+Once that's done, you'll need to adjust your Vite build command for when you're creating iOS builds. _Only_ for iOS
+builds. (If you try to run these builds on Android they probably won't work.)
+
+Add the `--mode=ios` to your build command. Run it before compiling your app for iOS. Here's an example using `npm`:
+
+```shell
+npm run build -- --mode=ios
+```
+
+## Compile your app
+
+To compile and run your app, simply run:
+
+```shell
+php artisan native:run --build=debug
+```
+
+This single command takes care of everything and allows you to run new builds of your application without having to
+learn any new editors or platform-specific tools.
+
+<aside class="relative z-0 mt-5 overflow-hidden rounded-2xl bg-pink-50 px-5 ring-1 ring-black/5 dark:bg-pink-600/10">
+
+#### Rule of thumb
+
+During development, keep `NATIVEPHP_APP_VERSION=DEBUG` to always refresh the Laravel application inside the native app.
+Your app will boot a little slower, but you will find that everything looks as you expect.
+
+It's better than scratching your head for an hour trying to figure out why your changes aren't showing!
+
+</aside>
+
+## Working with Xcode or Android Studio
+
+On occasion, it is useful to compile your app from inside the target platform's dedicated development tools, Android
+Studio and Xcode.
+
+If you're familiar with these tools, you can easily open the projects using the following Artisan command:
+
+```shell
+php artisan native:open
+```
+
+## Hot Reloading
+
+We've tried to make compiling your apps as fast as possible, but when coming from the 'make a change; hit refresh'-world
+of PHP development that we all love, compiling apps can feel like a slow and time-consuming process.
+
+So we've released hot reloading, which aims to make your development experience feel just like home. 
+
+You can enable hot reloading by running the following command:
+
+```shell
+php artisan native:watch {platform:ios|android}
+```
+
+This is useful during development for quickly testing changes without re-compiling your entire app. When you make
+changes to any files in your Laravel app, the web view will be reloaded and your changes should show almost immediately.
+
+### Implementation
+
+The proper way to implement this is to first `run` your app on your device/emulator, then start HMR with `npm run dev` then in a separate terminal run the `native:watch` command. This will reload any Blade/Livewire files as well as any recompiled assets (css/js etc).
+
+<aside class="relative z-0 mt-5 overflow-hidden rounded-2xl bg-pink-50 px-5 ring-1 ring-black/5 dark:bg-pink-600/10">
+
+#### Note
+
+ 
+
+```php
+'hot_reload' => [
```

Comment:
Especially here the background of the aside doesn't match well with the higlighted code block. Doubling the color confusion mentioned before